integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -173,57 +214,18 @@ tieredstore:
     high: 0.95
     low: 0.7
 
-# The domain hostPath for uuid mode in short-circuit
-domainHostPath: "/tmp/alluxio-domain"
-
-worker:
-  args:
-    - worker-only
-    - --no-format
-  # Properties for the worker component
-  properties:
-  resources:
-    limits:
-      cpu: "1"
-      memory: "2G"
-    requests:
-      cpu: "1"
-      memory: "2G"
-  ports:
-    rpc: 29999
-    web: 30000
-  # JVM options specific to the worker container
-  jvmOptions: ""
-  nodeSelector: {}
-  domainHostPath: "/tmp/alluxio-domain"
-  hostNetwork: true
-  dnsPolicy: ClusterFirstWithHostNet
-  ramdisk:
-    size: 1Gi
-  # shortCircuitPolicy can be local or uuid,
+# Short circuit related properties
+shortCircuit:
+  enabled: true
+  # The policy for short circuit can be "local" or "uuid",
   # local means the cache directory is in the same mount namespace,
   # uuid means interact with domain socket
-  shortCircuitPolicy: uuid
-
-jobWorker:
-  args:
-    - job-worker
-  # Properties for the jobWorker component
-  properties:
-  resources:
-    limits:
-      cpu: "1"
-      memory: "1G"
-    requests:
-      cpu: "1"
-      memory: "1G"
-  ports:
-    rpc: 30001
-    data: 30002
-    web: 30003
-  # JVM options specific to the jobWorker container
-  jvmOptions: ""
-
+  policy: uuid
+  pvcName: alluxio-worker-domain-socket
+  accessModes:
+    - ReadWriteOnce
+  size: 1Gi
```

Comment:
nit: rename to `pvcSize`. also this doesn't  need to be that large as its just a socket file, my guess is a default of `1Mi` should be fine